azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -115,6 +115,9 @@
   public static final String TOKEN_FILE_PERMISSIONS = "460";
   private static final String FS_HDFS_IMPL_DISABLE_CACHE =
       "fs.hdfs.impl.disable.cache";
+  // Some hadoop clusters have failover name nodes.
+  private static final String FS_FAILOVER_IMPL_DISABLE_CACHE =
```

Comment:
There might be other schemes in use here and cache has to be disabled for those as well. You would have to look at `fs.defaultFS` in Configuration and disable for the associated scheme as well.